pages/inspector/connect-inspector-to-gtm.mdx

```diff
@@ -1,14 +1,74 @@
-# Send data from Google Tag Manager to Avo Inspector
+import { Callout } from 'nextra/components';
 
-## Use Avo Inspector as a GTM tag
+# Google Tag Manager Source (beta)
 
-You can use a server side GTM to analyze your tracking with Avo Inspector.
+_This document describes how to send data from Google Tag Manager to Avo Inspector._
 
-We provide a tag template to import, which you can use to create a tag and connect it to your GTM clients.
+<Callout type="info">
+The Inspector Google Tag Manager integrations are currently in beta so please reach out to us at support@avo.app or in the chat bubble in the bottom right corner if you have any feedback.
+</Callout>
 
-> Avo Inspector operates on the environment level, please keep it in mind and create separate tags for development, staging and production data.
+## Destination Info
 
-## Guide to connect GTM and Avo Inspector
+* Supports both client side and server side Google Tag Manager  
+* Inspector events can be verified using the browser developer console
```

Comment:
@alex-tpom6oh how exactly can we do this? Maybe we want to specify the key needed to find them?